atom/browser/web_contents_preferences.cc

```diff
@@ -136,6 +136,17 @@ void WebContentsPreferences::AppendExtraCommandLineSwitches(
       LOG(ERROR) << "preload url must be file:// protocol.";
   }
 
+  // Custom args for renderer process
+  base::ListValue* customArgs;
+  std::string customArg;
+  if (web_preferences.GetList(options::kCustomArgs, &customArgs)) {
+    for (size_t i = 0; i < customArgs->GetSize(); i++) {
+      if (customArgs->GetString(i, &customArg)) {
+        command_line->AppendArg(customArg);
+      }
+    }
+  }
+
```

Comment:
`DictionaryValue::GetList` is deprecated, and the newer API wraps around `std::vector<base::Value>` so I _think_ the nondeprecated version would look something like:

```c++
// Custom args for renderer process
const base::Value* customArgs = web_preferences.FindPath(options::kCustomArgs);
if ((customArgs != nullptr) && (customArgs->is_list())) {
  for (const base::Value& customArg : customArgs->GetList()) {
    if (customArg.is_string()) {
      command_line->AppendArg(customArg.GetString());
    }
  }
}
```